library/src/lib/localizator-editor/localization-editor.component.html

```diff
@@ -0,0 +1,15 @@
+<fd-popover
+    [(isOpen)]="isOpen"
+    [closeOnEscapeKey]="closeOnEscapeKey" [closeOnOutsideClick]="closeOnOutsideClick"
+    [disabled]="disabled" [triggers]="triggers">
+    <fd-popover-control>
+        <ng-content select="fd-localization-editor-main"></ng-content>
+    </fd-popover-control>
+    <fd-popover-body>
+        <fd-menu>
+            <ul fd-menu-list class="fd-localization-editor__list">
+                <ng-content select="fd-localization-editor-item"></ng-content>
```

Comment:
Can you test this with a screenreader? Not sure this'll work since the list content isn't `<li>` elements

Reply:
It''s changed to be working with screen reader